verification.py

```diff
@@ -845,6 +845,14 @@ class MsmsSweeps(Test):
     use h=0.5 to calculate that for Aa.
     """
 
+    def _set_cmp_to_discoal(self, cmp_to_discoal):
```

Comment:
I'm not crazy about this pattern, modifying the state of the Test objects isn't a great idea for various reasons. Is there some reason we can't pass ``cmp_to_discoal`` to the calling function?